modules/phase_field/src/postprocessors/FeatureFloodCount.C

```diff
@@ -858,7 +1004,7 @@ FeatureFloodCount::appendPeriodicNeighborNodes(FeatureData & data) const
     {
       Elem * elem = _mesh.elemPtr(entity);
 
-      for (unsigned int node_n = 0; node_n < elem->n_nodes(); node_n++)
+      for (auto end = elem->n_nodes(), node_n = 0u; node_n < end; ++node_n)
```

Comment:
Does this mean that both end and node_n will be unsigned?  If not, what happens if `elem->n_nodes()` is not unsigned?  Does the `0u` just get cast to whatever the type of `elem->n_nodes()` is?

In general... I'm not a huge fan of this.  I would recommend splitting this up (save off `end` before entering the loop).